hardware/MQTT.cpp

```diff
@@ -3126,32 +3176,20 @@ void MQTT::InsertUpdateSwitch(_tMQTTASensor* pSensor)
 			{
 				szOnOffValue = "Set Level";
 				// Make level relative to 100.
-				if (
```

Comment:
Could you undo the formatting here. It makes the code harder to read

Reply:
Same again, reverted.